cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -88,6 +88,16 @@ function wait_for_istio_ready() {
     fi
 }
 
+# copy_istio_cni_conf_files copy the generated Istio CNI net conf file
+# (at '/etc/cni/multus/net.d/') to where Multus expect CNI net conf files ('/etc/cni/net.d/')
+function copy_istio_cni_conf_files() {
+    if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ] && [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
```

Comment:
what about `KUBVIRT_WITH_CNAO_SKIP_CONFIG` ?
The CR might be installed bit later and include the multus (but not sure if those folders do exists in this case at this stage ?)

can be ignored, if there is code that returns error upon this configuration

Reply:
>what about KUBVIRT_WITH_CNAO_SKIP_CONFIG ?
The CR might be installed bit later and include the multus (but not sure if those folders do exists in this case at this stage ?)

If by CR you mean CNAO CR, in the scenrtio of Multus deployment hangs no harm will do to the cluster if Istio confs will be copied to `/etc/cni/net.d`

Note that `copy_istio_cni_conf_files` is called only after both CNAO and Istio deployments are ready, so it is not likely to happen.

> This function is called only after both CNAO and Istio deployments are ready, so it is not likely to happen.

I am not sure what you mean, the error flag is set for script.

Reply:
I mean that the condition checks `KUBEVIRT_WITH_CNAO` but doesnt check the possible
`KUBVIRT_WITH_CNAO_SKIP_CONFIG`
That allows you to install CNAO but wihtout the CNAO CR (where you say which components to install)

So if CNAO will be installed without CR (i.e using `KUBVIRT_WITH_CNAO_SKIP_CONFIG`)
+ Istio will be installed
and later the CNAO CR will install multus, this WA code wont run and we will have the bug

Reply:
I am fine to not handle it but please don't remove it 
We are using KUBVIRT_WITH_CNAO_SKIP_CONFIG on KubeSecondaryDNS
Its helpful once you want to deploy the cached CNAO, but with your CR
This way you don't need to care about CNAO bumps, as it is part of the kubevirtci TAG that you need to bump anyhow